lib/rules/declaration-block-no-duplicate-properties/index.mjs

```diff
@@ -200,26 +201,31 @@ const rule = (primary, secondaryOptions, context) => {
 				const unprefixedDuplicatesAreEqual =
 					vendor.unprefixed(value) === vendor.unprefixed(duplicateValue);
 
+				/** @param {Declaration} node */
+				const fixer = (node) => {
+					node.remove();
+
+					return node.rangeBy({ index: 0, endIndex: 0 });
+				};
```

Comment:
[suggest] How about simplifying `fixer()` like this?

- Avoid `bind`
- Move outside the `eachDeclarationBlock` loop

```js
/** @param {Declaration} node */
const fixer = (node) => () => {
	node.remove();

	return node.rangeBy({ index: 0, endIndex: 0 });
};
```